superset-frontend/src/explore/components/controls/HiddenControl.tsx

```diff
@@ -17,29 +17,14 @@
  * under the License.
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Input } from 'src/common/components';
 
-const propTypes = {
-  onChange: PropTypes.func,
-  value: PropTypes.oneOfType([
-    PropTypes.string,
-    PropTypes.number,
-    PropTypes.object,
-    PropTypes.bool,
-    PropTypes.array,
-    PropTypes.func,
-  ]),
-};
-
-const defaultProps = {
-  onChange: () => {},
-};
+interface HiddenControlsProps {
+  onChange: () => void;
+  value: string | number | {} | boolean | [] | (() => void);
```

Comment:
I _think_ this should probably be

```suggestion
  value: string | number | Record<string, any> | boolean | any[] | Function;
```

Reply:
Thanks for the feedback. I have committed the suggested changes, and will keep them in mind for future.